crates/dprint/src/arg_parser.rs

```diff
@@ -380,6 +382,7 @@ EXAMPLES:
         .subcommand(
           Command::new("update")
             .about("Updates the plugins in the configuration file.")
+            .arg(Arg::new("yes").help("do not prompt to confirm to change url").short('y').long("yes").action(clap::ArgAction::SetTrue))
```

Comment:
We should change this message to be more clear about what they're agreeing to because there are security implications. Maybe "Upgrade process plugins without prompting to confirm checksums."